internal/events/event.go

```diff
@@ -419,9 +419,10 @@ func isDescending(order *string) (*bool, error) {
 }
 
 func escapePlaceHolders(message string) string {
-	escapedPercent := strings.ReplaceAll(message, "%", "\\%")
-	return strings.ReplaceAll(escapedPercent, "_", "\\_")
-
+	escapedBackslash := strings.ReplaceAll(message, "\\", "\\\\")
+	escapedUnderscore := strings.ReplaceAll(escapedBackslash, "_", "\\_")
+	escapedPercent := strings.ReplaceAll(escapedUnderscore, "%", "\\%")
+	return escapedPercent
```

Comment:
this one seems more easily maintainable:
```suggestion
	message = strings.ReplaceAll(message, "\\", "\\\\")
	message = strings.ReplaceAll(message, "_", "\\_")
	message = strings.ReplaceAll(message, "%", "\\%")
	return message
```